### PR TITLE
storage_benchmark: Clean the env files if failed to install benchmark

### PR DESCRIPTION
**Pull request by YongxueHong**
ID: 1708895
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
@lolyu  @hereischen  @XueqiangWei  @CongLi  @kimi1978wy  @ngu-niny  if you are free, please help me reiview, thanks.

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by xuwei@redhat.com

File: provider/storage_benchmark.py [hunk 1]
```diff
@@ -227,6 +227,17 @@ def clean(self, timeout=1800, force=False):
         if self.env_files:
             self._remove_env_files()
 
+    @classmethod
```
Comment (lolyu) [inline provider/storage_benchmark.py]: I believe `staticmethod` is better than `classmethod`

File: provider/storage_benchmark.py [hunk 2]
```diff
@@ -227,6 +227,17 @@ def clean(self, timeout=1800, force=False):
...
+    def _clean_env(cls, func):
+        """ Decorator that clean the env files. """
+        def __clean_env(self, *args):
```
Comment (lolyu) [inline provider/storage_benchmark.py]: better to use `functools.wraps` to decorate `__clean_env`, also, it is better defined as `__clean_env(self, *args, **kargs)`

File: provider/storage_benchmark.py (hunk 1 above)
Comment (YongxueHong) [reply provider/storage_benchmark.py]: Thanks, have updated.

File: provider/storage_benchmark.py (hunk 2 above)
Comment (YongxueHong) [reply provider/storage_benchmark.py]: Thanks, have updated.

**Comment by CongLi** [comment]
Ack.

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**